pkg/plugin/aggregation/run.go

```diff
@@ -190,9 +192,27 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 		return errors.Wrapf(err, "couldn't get aggregator pod")
 	}
 
-	for _, p := range plugins {
-		logrus.WithField("plugin", p.GetName()).Info("Running plugin")
-		go aggr.RunAndMonitorPlugin(context.Background(), time.Duration(cfg.TimeoutSeconds)*time.Second, p, client, nodes.Items, cfg.AdvertiseAddress, certs[p.GetName()], aggregatorPod, progressPort, pluginResultsDir)
+	orderedPlugins := getOrderedPlugins(plugins)
+
+	fmt.Printf("Received plugin launch order:")
```

Comment:
Sorry if I wasn't clear. In this situation (on the aggregator) be sure to be consistent with the existing code and use `logrus.Info("...")` rather than `fmt.Print` statements.

Either extract this into its own method and change to `logrus...` or have that new method return the string you want logrus to log for you.